lib/java_buildpack/framework/dynatrace_one_agent.rb

```diff
@@ -143,6 +172,10 @@ def host_id?
         @application.environment.key?(DT_HOST_ID)
       end
 
+      def skip_errors?
+        'true'.casecmp(credentials[SKIP_ERRORS] || 'false').zero?
```

Comment:
We've got a `to_b` method that will convert that string to a `boolean` value.  I'll update in a following commit.